src/utils/utils.ts

```diff
@@ -122,3 +123,52 @@ export function isNode() {
     typeof process.versions.node !== 'undefined'
   );
 }
+
+/**
+ * Convert a base64 string to a Uint8Array for Node.js and browser environments.
+ * @returns A Uint8Array
+ */
+export function base64ToBytes(base64String: string) {
+  if (isNode()) {
+    return new Uint8Array(Buffer.from(base64String, 'base64'));
+  }
+  // eslint-disable-next-line no-undef
```

Comment:
```suggestion
  /* eslint-env browser */
```
FYI the reason we see an error here by default is because we specify our global eslint environment as [`shared-node-browser`](https://github.com/algorand/js-algorand-sdk/blob/72653522835914c451357db9c1f98eb79ddbdf2a/.eslintrc.js#L4) -- only the definitions common to both environments are allowed by default.

Since this code branch should only get executed in the browser, we can make eslint aware of this with this comment, which is safer than ignoring all `undef` errors.